ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1316,29 +1154,95 @@ peer_filters:
         match: "< match as string >"
 ```
 
-### Router BGP Configuration
+### Routing
+
+### Service Routing Protocols Model
```

Comment:
missing a "#" so it appears under section routing.

Reply:
Good catch. Fixed with commit 5951187